test/gtest/ucs/test_topo.cc

```diff
@@ -14,42 +14,31 @@ class test_topo : public ucs::test {
 
 UCS_TEST_F(test_topo, find_device_by_bus_id) {
     ucs_status_t status;
-    ucs_sys_device_t unknown_dev1;
-    ucs_sys_device_t unknown_dev2;
-    ucs_sys_bus_id_t dummy_bus_id1;
+    ucs_sys_device_t current_index;
+    ucs_sys_device_t unknown_dev;
```

Comment:
the names current_index and unknown_dev seem wrong
maybe it should be just dev1, dev2?